packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -539,19 +548,37 @@ export class StripeHandler {
         }
       }
 
+      const taxOptions = this.automaticTax
+        ? { automatic_tax: taxSubscription }
+        : { taxRateId };
+
       const subscription: any =
         await this.stripeHelper.createSubscriptionWithPMI({
           customerId: customer.id,
           priceId,
           paymentMethodId,
           promotionCode: promotionCode,
-          taxRateId,
+          ...taxOptions,
         });
 
+      if (this.automaticTax && !taxSubscription) {
+        this.log.warn(
+          'subscriptions.createSubscriptionWithPMI.automatic_tax_failed',
+          {
+            uid,
+            automatic_tax: customer.tax?.automatic_tax,
+          }
+        );
+      }
```

Comment:
Should we report to Sentry if `automatic_tax` is `'not_collecting'`?

From the ticket:
> Should the create response with tax expanded indicate that we cannot tax them due to a missing registration, we will report it via Sentry with enough information to inform Finance what tax registration needs to be added.

Reply:
Unfortunately it's not easy to do unique queries across Stripe, so having lots of these pile-up there is not terribly actionable. It's easier to pull the data out of BigQuery, which is why the switch was made here on where the output goes.